pros/src/sensors/imu.rs

```diff
@@ -0,0 +1,326 @@
+use pros_sys::{PROS_ERR, PROS_ERR_F};
+use snafu::Snafu;
+
+use crate::error::{bail_on, map_errno, PortError};
+
+#[derive(Default, Debug, Clone, Copy, PartialEq, Eq)]
+pub struct InertialStatus {
+    pub raw: pros_sys::imu_status_e_t,
```

Comment:
I'm not sure that raw should be public. Usually we keep all pros-sys stuff private in the pros crate.